todo_app/app.py

```diff
@@ -1,44 +1,57 @@
 from flask import Flask, render_template, redirect, url_for, request
 
+from typing import cast
+
 from todo_app.data import trello_items
 
 from todo_app.flask_config import Config
 
 from todo_app.item import Item
 
-app = Flask(__name__)
-app.config.from_object(Config())
+from todo_app.view_model import ViewModel
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+
+    @app.route("/")
+    def index():
+        lists = [
+            (
+                cast(str,list["id"]),
+                cast(str,list["name"]),
+                [Item.from_trello_card(card) for card in list["cards"]],
+            )
+            for list in trello_items.get_lists()
+        ]
```

Comment:
I feel like this is complicated enough to merit our own `TrelloList` class or similar - expecting people to know that `list[1]` means the list's name seems unreasonable!